index.html

```diff
@@ -1,42 +1,42 @@
-<!DOCTYPE html>
-<html>
-
-<head>
-    <meta charset="UTF-8">
-    <title>Document</title>
-    <link rel="stylesheet" href="styles.css">
-</head>
-
-<body>
-    <table class="table">
-        <thead>
-            <tr>
-                <th>Full Name</th>
-                <th>Favourite Artist</th>
-                <th>State of Origin</th>
-                <th>Number of Siblings</th>
-                <th>Gender</th>
-            </tr>
-        </thead>
-        <tbody>
-            <!--           first person -->
-            <tr>
-                <td>Firstname Lastname</td>
-                <td>Artist Name</td>
-                <td>State Name</td>
-                <td>Number of Siblings</td>
-                <td><a class="btn btn__male">Male</a></td>
-            </tr>
-            <!--            second person -->
-            <tr>
-                <td>Firstname Lastname</td>
-                <td>Artist Name</td>
-                <td>State Name</td>
-                <td>Number of Siblings</td>
-                <td><a class="btn btn__female">Female</a></td>
-            </tr>
-        </tbody>
-    </table>
-</body>
-
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <table class="table">
+        <thead>
+            <tr>
+                <th>Full Name</th>

```

Comment:
Hi @tundealabi are close, well done. However, because another PR was merged into the master branch while your's was waiting you would need to run git fetch to have the latest version of the file on your machine.

After doing this, you should review if there's any. conflict in the file, fix them and resubmit your PR.